styleguide/source/assets/js/main-navigation-dr-finder.js

```diff
@@ -0,0 +1,230 @@
+(function ($, Drupal) {
```

Comment:
Do we need all this for simple drop-downs?

Reply:
This main-navigation-dr-finder.js has been removed